### DISCUSSION TITLE
Repeating same code

### DISCUSSION THREAD

**Discussion by PriyanshuVar** [community/community#8182] [Copilot Conversations]

**Comment by alt-art** [comment]
I have the same issue when you are programming the suggestion tries to rewrite the code below, which has already been implemented.

**Reply by alt-art** [reply]
When using JSX, the code gets worse.

**Reply by imranbaloch** [reply]
I am facing same issue where comments are keep repeating like below,
https://user-images.githubusercontent.com/667961/147380311-8b44c42d-4b73-4799-8f62-b6b0d5017d11.mp4

**Reply by Titanium-ctrl** [reply]
I have the same issue as both of you from time to time, where Copilot does nothing other than repeat previously written code, sometimes under a different function name, or comments
